scripts/validate-readme-toc.js

```diff
@@ -0,0 +1,19 @@
+// TODO: remove deep import once API is exposed (It is merged to master but not published yet)
+const transform = require('doctoc/lib/transform');
+const {readFileSync} = require('fs');
+const path = require('path');
+
+const README_PATH = path.join(__dirname, '../README.md');
```

Comment:
In the script you passed the file path in, is the plan to expand this to allow for using that instead of the constant?